src/state_machine.zig

```diff
@@ -3866,6 +3866,33 @@ test "create_transfers: balancing_debit | balancing_credit (amount=0)" {
     );
 }
 
+test "create_transfers: balancing_debit | balancing_credit (amount=0, balance≈maxInt)" {
+    try check(
+        \\ account A1  0  0  0  0  _  _  _ _ L1 C1   _ D<C   _ _ _ _ ok
+        \\ account A2  0  0  0  0  _  _  _ _ L1 C1   _ D<C   _ _ _ _ ok
+        \\ account A3  0  0  0  0  _  _  _ _ L1 C1   _ D<C   _ _ _ _ ok
+        \\ account A4  0  0  0  0  _  _  _ _ L1 C1   _ D<C   _ _ _ _ ok
+        \\ commit create_accounts
+        \\
+        \\ setup A1 0  0 0 -1
+        \\ setup A4 0 -1 0  0
+        \\
+        \\ transfer   T1 A1 A2    0   _  _  _  _    _ L1 C1   _   _   _   _ BDR   _  _ _ ok
+        //\\ transfer   T2 A3 A4    0   _  _  _  _    _ L1 C1   _   _   _ BDR   _   _  _ _ ok
```

Comment:
Is it supposed to be commented?

Reply:
:facepalm: No, I meant to add that back once I got the first part working, but forgot. Thanks!